pysat/_instrument.py

```diff
@@ -498,6 +502,7 @@ def __setitem__(self, key, new):
             self.meta[key] = new
             
         else:
+            """
```

Comment:
Are these string sections meant to maintain old code as comments?  This isn't necessary since github stores the old versions of the code.  It also makes side-by-side comparisons (such as the one I'm doing here) harder.